cloudv2/test/counter/dec.test.w

```diff
@@ -0,0 +1,47 @@
+bring "../../" as cloud;
```

Comment:
are these basically the tests from `sdk_tests`?

Let's make sure to delete them once we've migrated.

Reply:
Yep, pretty much copied fully